puppet/modules/web/files/deploy-yumrepo.sh

```diff
@@ -73,7 +73,7 @@ purgecdn() {
 	set +f
 	for d in "${REPO_PATH}"/*; do
 		purge_base="https://yum.theforeman.org/${REPO_DEST}/$(basename $d)"
-		fastly-purge ${purge_base} foreman-release.rpm foreman-client-release.rpm
+		fastly-purge ${purge_base} foreman-release.rpm foreman-client-release.rpm katello-repos-latest.rpm
```

Comment:
I haven't tested this, but does it work with katello and its subdirectories? or is it `katello/...`

Reply:
yeah, the purge base is what makes the magic :)